app/grandchallenge/challenges/models.py

```diff
@@ -75,6 +75,11 @@ def filter_tag(self):
 class ImagingModality(models.Model):
     """ Stores the modality options, eg, MR, CT, PET, XR """
 
+    MODALITY_OCT = "OCT"  # Optical coherence tomography
+    MODALITY_CF = "CF"  # Color fundus photography
+    MODALITY_FA = "FA"  # Fluorescein angiography
+    MODALITY_IR = "IR"  # Infrared Reflectance imaging
+
```

Comment:
That's correct. I added this because I needed a way to reference the different modalities consistently. I did try to always use get_or_create() in the importers so the modalities should be on the database. I can also just add the declarations to settings and create a datamigration to make sure they exist. Should I just do that or do you have a better suggestion?

Reply:
I moved the modality declarations to settings and referenced it from there. Changed the values to the already existing values on grandchallenge. Also, made sure every time it is referenced in importers that it is done by `get_or_create` so it has to exist. (2864b9cd4e744392cc213e61a52a2e94816d3284)